### ISSUE TITLE
断点续训

### ISSUE THREAD

**Issue by li-25-creater** [BICLab/SpikeYOLO#51]
您好，很抱歉打扰，我想问一下您代码的断电续训是怎么做的呢？是这样的吗？
model =YOLO("train/weights/last.pt")
model.train(data="VOC-test.yaml",device=[0],epochs=100,resume = True)

**Comment by baekchaesun** [comment]
I am not the author, but I am providing an answer on their behalf for a quick resolution.

The code you mentioned should work fine.

**Comment by li-25-creater** [comment]
@baekchaesun Thank you very much for answering my question, but I encountered some issues while using this method. Although I was able to continue training with the previous round using this method, the map and loss values still started training from the first round. I am really troubled by this problem

**Comment by li-25-creater** [comment]
@baekchaesun Thank you very much for your answer!
 I have already solved this problem!

**Comment by arsasi** [comment]
> [@baekchaesun](https://github.com/baekchaesun) Thank you very much for your answer! I have already solved this problem!

请问你怎么解决的，使用load和resume我依然是从第一轮开始训练，然后学习率也是初始值。

**Comment by li-25-creater** [comment]
@arsasi  具体方法是完全参考yolov8官方的文档，但我跑的时候这样续训的轮数是续上去了但loss什么的还是从第一轮开始，我这个问题不常见，可能是我自己的问题，你可以参考yolov8的官方文档进行断点续训，若出现和我相同的问题，请给我留言
